addon-resizer/README.md

```diff
@@ -5,7 +5,11 @@ vertically scales the dependent container up and down. Currently the only
 option is to scale it linearly based on the number of nodes, and it only works
 for a singleton.
 
-Currently recommended version is 1.8, on addon-resizer-release-1.8 branch.
```

Comment:
Can we keep this line, ideally bolded, prepend with a "NOTE: " and then link to the 1.8 readme:
See [1.8 documentation for details](https://github.com/kubernetes/autoscaler/blob/addon-resizer-release-1.8/addon-resizer/README.md)

(The readme in that branch is a bit outdated but I'll try to fix that right away)